index.js

```diff
@@ -51,9 +52,19 @@ function handleFinish(vulnerabilities, logData = '', configs = {}) {
     console.info(toDisplay);
   }
 
+  // Display the unused exceptionId's
+  const exceptionsIdsAsArray = Array.isArray(exceptionIds) ? exceptionIds : [exceptionIds];
```

Comment:
Hmm this still does not work correctly because `vulnerabilities` is the list of `vulnerabilities` after the exceptions are already filtered, I need to put in the unfiltered exceptions.